.gitignore

```diff
@@ -2,3 +2,4 @@ __pycache__/
 *.py[cod]
 ncvoter/downloads/
 deployment/.vault_pass
+downloads
```

Comment:
I think we could probably remove line 3 now (since this one takes precedence over that one in all cases)